docker/docker-entrypoint.sh

```diff
@@ -12,7 +12,7 @@ fi
 # to the "dispatch" binary. Other parameters are passed through
 # Example: "./docker-entrpoint.sh server start" becomes "dispatch server start"
 case "${@:1:2}" in
-	"server start" | "database upgrade" | "database downgrade" | "scheduler start" )
+	"server start dispatch.main:app" | "database upgrade" | "database downgrade" | "scheduler start" )
```

Comment:
I think this will cause the match to fail. This only joins 2 tokens in bash: `"${@:1:2}"` 

The app argument is still passed through at call-time though. See https://github.com/Netflix/dispatch-docker/pull/12/files#diff-4e5e90c6228fd48698d074241c2ba760R22

This bash script is just matching on presence of "server start" and then preprending `dispatch` to the arguments. The "@" symbol means the original arguments are all added on.

I don't think this PR is needed and can probably be closed.